modules/template/src/main/twirl/scaladex/view/search/result.scala.html

```diff
@@ -29,19 +28,19 @@ <h4 class="text-center">All the projects below have beginner-friendly issues whi
   <div class="col-md-2">
     <div class="result-count">
       @if(pagination.current == 1) {
-        @pagination.itemCount results
+        @pagination.pageSize results
       } else {
-        Page @pagination.current of @pagination.itemCount results
+        Page @pagination.current of @pagination.pageSize results
```

Comment:
so pageSize is the entire result size ?

Reply:
no its the size of the page

Reply:
Usually it is 20 except if this is the last page and there is less than 20 items in it